.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.1.2rc4
+current_version = 2.2.0
```

Comment:
We could first release a stable version of 2.1.2 for the fixes in that.

With all these removals I'd suggest we make a 2.2.0rc1 first to get some feedback from it in development?

Reply:
Ill create a 2.1.2, I would like to go straight to 2.2.0 if possible as it only handles removals. In other words, lets test and then release 2.2.0 straightaway. Keeps 2.2.0 nice and clean